server/src/main/resources/org/opensearch/bootstrap/security.policy

```diff
@@ -194,4 +199,8 @@ grant {
   permission java.io.FilePermission "/sys/fs/cgroup/cpuacct/-", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/memory", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/memory/-", "read";
+
```

Comment:
Why do we need these permissions here?

Reply:
`arrow-memory` which is part of lib needs them. Since we can't add security permissions on libs, so added them here. Do you any concerns?

Reply:
It should apply to `grant codeBase "${codebase.opensearch-arrow-memory-shaded}" {`

Reply:
done